www/on/npm/%package/index.html.spt

```diff
@@ -27,4 +27,15 @@ url = 'https://npmjs.com/package/' + package.name
 {% endblock %}
 
 {% block content %}
+{% if user.ANON %}
+<h2>{{ _( '{npm_package} has not been claimed on Gratipay.'
+        , npm_package='npm/' + package_name
```

Comment:
Seems it's not highlighted as link in screenshot. But I don't want to block us on this.